requirements.txt

```diff
@@ -5,5 +5,5 @@ numpy>=1.21.4
 python_speech_features>=0.6
 scipy>=1.10.1
 pytsmod>=0.3.7
-PySimpleGUIWx==0.17.2; sys_platform == 'windows'
-PySimpleGUIQt>=0.35.0; sys_platform != 'windows'
+PySimpleGUIWx==0.17.2; platform_system == 'Windows'

```

Comment:
This change seems unnecessary?  Also, the old version is consistent with the example in this documentation:
https://pip.pypa.io/en/stable/reference/requirement-specifiers/

Reply:
sys_platform would need to match 'win32', not 'Windows', I switched it to be consistent with the platform check in the code which uses platform.system() instead of sys.platform()

Reply:
I am fine switching this back to sys_platform, but we should keep platform checks consistent between dependencies and the code, up to you.

Reply:
So long as platform_system works, I think it's more clear so I'm fine with changing it.